### PR TITLE
AST-3196 - Release master PR for 4.1.5

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Sub tasks under this task going to release in v4.1.5
- https://brainstormforce.atlassian.net/browse/AST-3196

### Types of changes
- Bug fix 

### How has this been tested?
- As per the individual tasks added in the list

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php [hunk 1]
```diff
@@ -253,10 +253,10 @@ function astra_off_canvas_static_css() {
 		.ast-mobile-popup-drawer .ast-mobile-popup-inner {
 			width: 100%;
 			transform: translateX(-115%);
-			max-width: 0;
+			max-width: 90%;
 			left: 0;
 			top: 0;
-			background: transparent;
```
Comment (nathanrodrigues2111) [inline inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php:259]: Can this be taken from our variable color pallet , If we have something similar rather then making it static

File: inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php [hunk 2]
```diff
@@ -342,10 +342,10 @@ function astra_off_canvas_static_css() {
 		.ast-mobile-popup-drawer .ast-mobile-popup-inner {
 			width: 100%;
 			transform: translateX(100%);
-			max-width: 0;
+			max-width: 90%;
 			right: 0;
 			top: 0;
-			background: transparent;
```
Comment (nathanrodrigues2111) [inline inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php:348]: Same as above

File: inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php (hunk 1 above)
Comment (imnavanath) [reply inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php:259]: Its just reverting the original PR. 

https://github.com/brainstormforce/astra/pull/5034 So for safer side just reverting back to stable code.

File: inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php (hunk 2 above)
Comment (imnavanath) [reply inc/builder/type/header/off-canvas/dynamic-css/dynamic.css.php:348]: Its just reverting the original PR. 

https://github.com/brainstormforce/astra/pull/5034 So for safer side just reverting back to stable code.

**Pull request merged**